docker/cygnus-ngsi/Dockerfile

```diff
@@ -139,6 +141,20 @@ ENV CYGNUS_POSTGRESQL_BATCH_TIMEOUT ""
 ENV CYGNUS_POSTGRESQL_BATCH_TTL ""
 ENV CYGNUS_POSTGRESQL_ENABLE_CACHE ""
 
+# SlicingDice options
```

Comment:
New env vars should be documented in the proper documentation file (more on this in a general comment I'll provide soon).